src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -68,13 +86,12 @@ export const StyledLink = styled(Link)`
   }
 `
 
-// TODO: add background url when supported by Kaws
-const ImgPlaceholder = styled.div`
-  width: 85px;
+export const ArtworkImage = styled.img<{ width: number }>`
+  width: ${({ width }) => width}px;
   height: 125px;
   background-color: ${color("black10")};
-  background-position: center;
-  background-size: cover;
+  object-fit: cover;
```

Comment:
nice!

Reply:
Changed this from a background image to an actual image, based on a suggestion from @joeyAghion that including image `alt` could be an SEO win.